### ISSUE TITLE
after 2.4.5 update e2guardian not loging.. 

### ISSUE THREAD

**Issue by serlogo53** [marcelloc/Unofficial-pfSense-packages#79]
hi marcelloc,
after pfsense update logging not working. 
status look like running but noting log.
![e2 error](https://user-images.githubusercontent.com/14811860/93435149-aea31c80-f8d1-11ea-97cf-cc362a5e19a3.JPG)

**Comment by serlogo53** [comment]
after remove and install again. i show 

/usr/local/etc/rc.d/e2guardian.sh: WARNING: failed to start e2guardian 
Error parsing the e2guardian.conf file or other e2guardian configuration files

**Comment by forid786** [comment]
Hi @serlogo53 

Have you tried clearing the log and then restarting E2 Guardian?

Regards, 
Forid

**Comment by serlogo53** [comment]
yes but nothing.. before update when working e2 guardian: on the firewall connection status table 
lan ip -> 127.0.0.1 

now :
lan ip -> wan ip (xx.xx.x.xx)

**Comment by serlogo53** [comment]
i am using e2guardian direct connect mode.

**Comment by marcelloc** [comment]
Try to run it in foreground mode under console to see if it show any info/error 

/usr/local/sbin/e2guardian -QN

**Comment by serlogo53** [comment]
![2](https://user-images.githubusercontent.com/14811860/93561721-61848080-f98d-11ea-85e0-1fba643d9731.JPG)
![1](https://user-images.githubusercontent.com/14811860/93561727-634e4400-f98d-11ea-818f-291d96a19847.JPG)
